libpod/container_internal_linux.go

```diff
@@ -1000,7 +1000,7 @@ func (c *Container) generateResolvConf() (string, error) {
 	nameservers := resolvconf.GetNameservers(resolv.Content)
 	// slirp4netns has a built in DNS server.
 	if c.config.NetMode.IsSlirp4netns() {
-		nameservers = append(nameservers, "10.0.2.3")
+		nameservers = append([]string{"10.0.2.3"}, nameservers...)
```

Comment:
Should we check ig nameservers is 127.0.0.1 And then shift?  Or is this a safe default?  
Is 10.0.2.3 something that causes slirp4netns to forward to the local networks DNS?

Reply:
yes, 10.0.2.3 is managed directly by slirp4netns and then it is resolved by the main process running in the host network namespace.